define-your-target-audience/readme.md

```diff
@@ -0,0 +1,37 @@
+>Goal of this guide: understanding your target audience for your landing page
+
+# How to target the right audience
+
+Knowing who a website’s visitors are (or will be) is crucial for designers and copywriters to know, because it helps us make all of the decisions that will ultimately comprise the website we deliver.
+
+This means stepping outside of our role as writers and designers and putting on a marketer’s cap, because ultimately that’s what this information is all about. Marketers focus on helping clients *communicate effectively with their customers*, and that’s exactly what a successful website does.
+
```

Comment:
"Marketers focus on helping clients..."

Clients not entirely clear. Do marketers always get hired externally?